printing/printing.py

```diff
@@ -47,14 +47,11 @@ def convert_to_pdf(filename):
     else:
         raise ValueError()
 
-    subprocess.check_call(
-        [SANDBOX_PATH, tmpdir, 'gs', '-sDEVICE=pdfwrite', '-dNOPAUSE',
-         '-dBATCH', '-dSAFER', '-dCompatibilityLevel=1.4',
-         '-sOutputFile=' + tmpdir + '/final.pdf', out])
-
+    subprocess.check_call(['./sandbox.sh', tmpdir, 'gs', '-sDEVICE=pdfwrite', '-dNOPAUSE', '-dBATCH', '-dSAFER', '-dCompatibilityLevel=1.4',
```

Comment:
Why not `SANDBOX_PATH`? It seems that you have a variable for that.
```suggestion
    subprocess.check_call([SANDBOX_PATH, tmpdir, 'gs', '-sDEVICE=pdfwrite', '-dNOPAUSE', '-dBATCH', '-dSAFER', '-dCompatibilityLevel=1.4',
```